content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx

```diff
@@ -537,7 +537,7 @@ In this guide, you'll use [Prisma Migrate](/concepts/components/prisma-migrate)
 
 ```prisma file=prisma/schema.prisma copy
 model Post {
-  id        BigInt   @id @default(autoincrement())
+  id        BigInt   @id @default(sequence())
```

Comment:
Does this change take the follow up differences in generated SQL or similar (if the getting started has this) into account?

Reply:
Yes. Generated sql uses sequence instead of autoincrement.